app/components/agenda/agendaitem/decision/editor.hbs

```diff
@@ -23,6 +23,7 @@
   </div>
   <WebComponents::VlRdfaEditor
     data-test-agendaitem-decision-rdfa-editor
+    @size={{@size}}
```

Comment:
This file was removed in a recent PR that was just merged. It was changed to the SayEditor.

So this change will need to be applied to the SayEditor, after a pull of the `development` branch

Reply:
@tdn Was thinking at the moment I merged from `development`: "Huh? where did my file go 😅", but it was an easy fix once I found the correct component/file. Thanks for the warning!